lib/lrmd/lrmd_alerts.c

```diff
@@ -130,6 +130,9 @@ exec_alert_list(lrmd_t *lrmd, GList *alert_list, enum crm_alert_flags kind,
     bool any_success = FALSE, any_failure = FALSE;
     const char *kind_s = crm_alert_flag2text(kind);
     crm_time_hr_t *now = NULL;
+    crm_time_t *dt = crm_time_new(NULL);
+    char *timestamp_epoch = calloc(1, 34);
+    char *timestamp_nanos = calloc(1, 14);
```

Comment:
These are small enough to use arrays rather than calloc()

Reply:
I change the use of local arrays.

char timestamp_xxx[34];
char timestamp_xxx[14];